apps/right-to-rent-check/index.js

```diff
@@ -5,13 +5,27 @@ const controllers = require('hof-controllers');
 module.exports = {
   name: 'right-to-rent-check',
   baseUrl: '/right-to-rent-check',
+  params: '/:action?/:id?',
   steps: {
     '/': {
       controller: controllers.start,
```

Comment:
no, we can just lose the '/' step - if it isn't there the default behaviour is to redirect to the first step when '/' is hit. Also we aren't clearing the session, that was stupid and defied the point of session :)